lib/database_connection.py

```diff
@@ -73,6 +73,6 @@ def _database_name(self):
 def get_flask_database_connection(app):
     if not hasattr(g, 'flask_database_connection'):
         g.flask_database_connection = DatabaseConnection(
-            test_mode=os.getenv('APP_ENV' , 'test') == 'test')
+            test_mode=os.getenv('APP_ENV' , 'test') == 'test') # to populate the development table via app: test_mode=os.getenv('APP_ENV' , 'test') == 'development')
```

Comment:
Might have to chat this one through

Reply:
Removed comment, and clarified with a need for a .env file and import of library to change environment.